packages/client/hmi-client/src/components/widgets/tera-related-publications.vue

```diff
@@ -147,4 +146,24 @@ const sendForEnrichments = async (/* _selectedResources */) => {
 ul {
 	margin: 1rem 0;
 }
+
+.no-artifacts {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.no-artifacts-img {
+	width: 30%;
+	padding: 10px;
+}
+
+.no-artifacts-text {
```

Comment:
You don't need specialize class when the parent as one.
`.no-artifacts img` would suffice

Reply:
@YohannParis I know.  I personally like to use a dedicated class vs doing something like that.  honestly don't know what's actually the preferred way.